src/common/snippets/src/lowered/pass/define_buffer_clusters.cpp

```diff
@@ -320,8 +320,6 @@ bool DefineBufferClusters::are_buffer_neighbours(const ExpressionPtr& up, const
 
 void DefineBufferClusters::parse_memory_access_op(const ExpressionPtr& expr) {
     const auto ma = std::dynamic_pointer_cast<modifier::MemoryAccess>(expr->get_node());
-    if (!ma->is_full_memory_access_op(expr->get_node()))
-        return;
```

Comment:
The details were shared offline: this outdated check is not needed anymore since now such cases are handled through generic algorithm (`parse_memory_access_op`).